applications/StructuralMechanicsApplication/custom_strategies/custom_schemes/explicit_central_differences_scheme.hpp

```diff
@@ -0,0 +1,922 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics 
+//
+//  License:		 BSD License 
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Klaus B Sautter (based on the work of MSantasusana)
+//					 
+//
+
+#if !defined(KRATOS_EXPLICIT_CENTRAL_DIFFERENCES_SCHEME_HPP_INCLUDED)
+#define  KRATOS_EXPLICIT_CENTRAL_DIFFERENCES_SCHEME_HPP_INCLUDED
+
+
+/* System includes */
+
+
+/* External includes */
+//#include "boost/smart_ptr.hpp"
+
+
+/* Project includes */
+#include "includes/define.h"
+#include "includes/model_part.h"
+#include "solving_strategies/schemes/scheme.h"
+#include "includes/variables.h"
+#include "containers/array_1d.h"
+
+namespace Kratos
+{
+
+  /**@name Kratos Globals */
+  /*@{ */
+
+
+  /*@} */
+  /**@name Type Definitions */
+  /*@{ */
+
+  /*@} */
+
+
+  /**@name  Enum's */
+  /*@{ */
+
+
+  /*@} */
+  /**@name  Functions */
+  /*@{ */
+
+
+
+  /*@} */
+  /**@name Kratos Classes */
+  /*@{ */
+
+  template<class TSparseSpace,
+	   class TDenseSpace //= DenseSpace<double>
+	   >
+  class ExplicitCentralDifferencesScheme : public Scheme<TSparseSpace,TDenseSpace>
+  {
+
+  public:
+    /**@name Type Definitions */
+    /*@{ */
+    KRATOS_CLASS_POINTER_DEFINITION( ExplicitCentralDifferencesScheme );
+
+    typedef Scheme<TSparseSpace,TDenseSpace> BaseType;
+
+    typedef typename BaseType::TDataType TDataType;
+
+    typedef typename BaseType::DofsArrayType DofsArrayType;
+
+    typedef typename BaseType::TSystemMatrixType TSystemMatrixType;
+
+    typedef typename BaseType::TSystemVectorType TSystemVectorType;
+
+    typedef typename BaseType::LocalSystemVectorType LocalSystemVectorType;
+
+    typedef typename BaseType::LocalSystemMatrixType LocalSystemMatrixType;
+    
+    typedef ModelPart::ElementsContainerType ElementsArrayType;
+
+    typedef ModelPart::ConditionsContainerType ConditionsArrayType;
+    
+    typedef ModelPart::NodesContainerType NodesArrayType;
+
+    ExplicitCentralDifferencesScheme(
+				     const double  rMaximumDeltaTime,
+				     const double  rDeltaTimeFraction,
+				     const double  rDeltaTimePredictionLevel,
+				     const bool    rRayleighDamping
```

Comment:
I think the Rayleighdamping is handeled differently, can you check in the harmonic strategy or the nodal concentrated element?